### DISCUSSION TITLE
How to using IF blocks

### DISCUSSION THREAD

**Discussion by estiarista** [postmanlabs/postman-flows#429] [General]
How to using IF blocks? if I run flows for **if blocks** and then if the result is true so for next block i want to take a parameter that successfully added only if the result is true to run next send request other API.
![image](https://user-images.githubusercontent.com/45717334/207902862-31ed91cd-5de2-4a3e-a55f-b7117d3487fa.png)

**Comment by minoseah629** [comment]
this page is an overly complicated example of an If block.
https://learning.postman.com/docs/postman-flows/flows-intro/building-your-first-flow/#using-an-if-block-to-branch-based-on-response-data.

Just provide a simple example of proper usage.
